browser/ui/BUILD.gn

```diff
@@ -184,7 +186,10 @@ source_set("ui") {
   }
 
   if (!is_android) {
-    deps += [ "//brave/app:brave_generated_resources_grit" ]
+    deps += [
+      "//brave/app:brave_generated_resources_grit",
+      "//brave/browser:version_info",
```

Comment:
if the deps for `brave_about_handler.cc` have `!is_android` then the source files also need to be in `!is_android`

Reply:
Yes, they are: https://github.com/brave/brave-core/pull/3828/files#diff-4298e412eb456ad3ceed2346abb95f6aL33